### PR TITLE
Protocol Guild Tenant

### PR DESCRIPTION
**Pull request by mcgingras**

### CODE REVIEW

File: src/app/proposals/draft/[id]/components/OwnerOnly.tsx [hunk 1]
```diff
@@ -11,19 +11,20 @@ const OnlyOwner = ({
...
+  //         <div className="mb-4 flex flex-row items-center space-x-6">
+  //           <h1 className="font-black text-primary text-2xl m-0">Loading...</h1>
+  //         </div>
+  //         <div className="grid grid-cols-1 sm:grid-cols-3 gap-y-4 sm:gap-y-0 gap-x-0 sm:gap-x-6">
+  //           <section className="col-span-1 sm:col-span-2 order-last sm:order-first h-36 bg-agora-stone-100 animate-pulse rounded-lg"></section>
+  //           <section className="col-span-1 h-12 bg-agora-stone-100 animate-pulse rounded-lg"></section>
+  //         </div>
+  //       </main>
+  //     );
+  //   }
```
Comment (andreitr) [inline src/app/proposals/draft/[id]/components/OwnerOnly.tsx:28]: Why are we commenting this out?

File: src/components/Metrics/DAOMetricsHeader.jsx [hunk 2]
```diff
@@ -72,21 +72,26 @@ export default function DAOMetricsHeader({ metrics }) {
...
-                    </HoverCardTrigger>
-                    <HoverCardContent
-                      className="w-full shadow"
-                      side="bottom"
-                      sideOffset={3}
-                    >
-                      <span>{token.symbol} currently delegated to a voter</span>
-                    </HoverCardContent>
-                  </HoverCard>
+                  {contracts.token.isERC20() && (
```
Comment (andreitr) [inline src/components/Metrics/DAOMetricsHeader.jsx:75]: nice!

**Review by andreitr — APPROVED** [review]
LGTM

File: src/app/proposals/draft/[id]/components/OwnerOnly.tsx (hunk 1 above)
Comment (mcgingras) [reply src/app/proposals/draft/[id]/components/OwnerOnly.tsx:28]: I left a comment  "causing jitter when loading... need to figure out a better long term solution"

I think the account was reconnecting on each stage transition in the form and it was jittering quite a bit

Comment (mcgingras) [reply src/app/proposals/draft/[id]/components/OwnerOnly.tsx:28]: still does the onlyOwner check, just doesn't try to show a loader

**Pull request merged**